src/main/java/de/zalando/aruha/nakadi/config/JsonConfig.java

```diff
@@ -33,14 +39,15 @@
     @Bean
     @Primary
     public ObjectMapper jacksonObjectMapper() {
-        final ObjectMapper objectMapper = new ObjectMapper()
-                .setPropertyNamingStrategy(CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+        final ObjectMapper objectMapper = new ObjectMapper();
 
+        objectMapper.setPropertyNamingStrategy(CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
         objectMapper.registerModule(enumModule());
+        objectMapper.registerModule(eventTypeSchemaModule());
```

Comment:
I think this features follows the *robustness principle* where we are flexible on the input side, as far as it's not ambiguous, but we are strict in the output format. Since this is even backward compatible, I would say that it's completely fine to merge 👍